packages/@lwc/engine-core/src/framework/check-version-mismatch.ts

```diff
@@ -38,7 +38,7 @@ export function checkVersionMismatch(
 ) {
     const versionMatcher = func.toString().match(LWC_VERSION_COMMENT_REGEX);
     if (!isNull(versionMatcher) && !warned) {
-        if (process.env.SKIP_LWC_VERSION_MISMATCH_CHECK === 'true') {
+        if (process?.env?.SKIP_LWC_VERSION_MISMATCH_CHECK === 'true') {
```

Comment:
Isn't this still gonna fail because the variable doesn't exist?

Reply:
Indeed! Need to use `typeof process` instead. TIL.